lib/classes/PluginManager.test.js

```diff
@@ -779,6 +779,64 @@ describe('PluginManager', () => {
     });
   });
 
+  describe('#parsePluginsObject()', () => {
+    const parsePluginsObject = (servicePlugins, expectedResult) => {
```

Comment:
You should name this helper function `parsePluginsObjectAndVerifyResult`. Then it is clear that you do not call the original `parsePluginsObject` function directly in the tests below, but combine verification and call in one function.

Reply:
Thank you, I have renamed the name to make it more clear.